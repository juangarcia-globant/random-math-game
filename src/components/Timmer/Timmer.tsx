type TimmerProps = {
  seconds: number;
};

export const Timmer: React.FunctionComponent<TimmerProps> = ({ seconds }) => {
  return (
    <span>
      <i className="fas fa-clock"></i> <b>{ seconds }</b>
    </span>
  )
}

export default Timmer;