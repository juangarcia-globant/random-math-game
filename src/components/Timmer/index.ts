export { default, Timmer } from './Timmer';