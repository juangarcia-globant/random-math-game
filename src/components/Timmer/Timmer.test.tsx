import { render, screen } from '@testing-library/react';
import Timmer from './Timmer';

describe("<Timmer />", () => {
  test("Matches the snapshot", () => {
    const { asFragment } = render(<Timmer seconds={10} />);
    expect(asFragment()).toMatchSnapshot()
  });

  test("Seconds should be rendered", () => {
    render(<Timmer seconds={20} />);
    expect(screen.getByText(20)).toBeInTheDocument();
  });
});