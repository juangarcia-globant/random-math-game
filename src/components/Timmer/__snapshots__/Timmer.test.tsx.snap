// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timmer /> Matches the snapshot 1`] = `
<DocumentFragment>
  <span>
    <i
      class="fas fa-clock"
    />
     
    <b>
      10
    </b>
  </span>
</DocumentFragment>
`;
